google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/FormatOptionsTest.java

```diff
@@ -30,23 +30,29 @@ public void testConstructor() {
     assertEquals(FormatOptions.JSON, options.getType());
     options = new FormatOptions(FormatOptions.DATASTORE_BACKUP);
     assertEquals(FormatOptions.DATASTORE_BACKUP, options.getType());
+    options = new FormatOptions(FormatOptions.AVRO);
+    assertEquals(FormatOptions.AVRO, options.getType());
   }
 
   @Test
+  @SuppressWarnings("deprecation")
```

Comment:
Can you change the `options.type()` calls to `options.getType()` here so that you don't need to suppress deprecation warnings?